Mathlib/Analysis/Asymptotics/Asymptotics.lean

```diff
@@ -1926,12 +1926,18 @@ theorem isLittleO_const_id_cobounded (c : F'') :
   isLittleO_const_left.2 <| .inr tendsto_norm_cobounded_atTop
 #align asymptotics.is_o_const_id_comap_norm_at_top Asymptotics.isLittleO_const_id_cobounded
 
-theorem isLittleO_const_id_atTop (c : E'') : (fun _x : ℝ => c) =o[atTop] id :=
-  isLittleO_const_left.2 <| Or.inr tendsto_abs_atTop_atTop
+theorem isLittleO_const_id_cocompact [ProperSpace F''] (c : E'') :
+    (fun _ => c) =o[cocompact F''] id :=
+  isLittleO_const_left.2 <| Or.inr tendsto_norm_cocompact_atTop
```

Comment:
While you're at it, could you please add a `cobounded` version? It doesn't need the `ProperSpace` assumption.

Reply:
Yes.